### ISSUE TITLE
Modify a toot without deleting it

### ISSUE THREAD

**Issue by stemy2** [mastodon/mastodon#13935]
### Pitch

For now, if i want to modify one of my toots, i have to delete and rewrite it, this is not very ergonomic.

### Motivation

I make often some typos and forget some elements in already published toots. I'd like to correct it without having to delete it.

**Comment by mstdn** [comment]
This is intended not to. If other users boost/favorite a toot and it becomes popular the author can edit the content that could be entirely different so the users who interacted with that original now boost maybe something that don't agree with them.

**Comment by stemy2** [comment]
Even in that case, you could add the mention "modified toot" with a list of modification, or disable the function as long nobody interacted with it.

**Comment by mstdn** [comment]
> Even in that case, you could add the mention "modified toot" with a list of modification, or disable the function as long nobody interacted with it.

Edit before interactions would be a good idea yes!

**Comment by Sasha-Sorokin** [comment]
Actually, not before the first interaction, but rather in one-five minutes interval and no major changes. I remember hearing that it was really common for the people to notice typos only after they posted something, not before, even when they check what they wrote before posting, but seems I couldn't find it to link here; there even was a term for that!

Anyway, after time runs out, edit option gets replaced by redraft one, seems fair. Also, from UX note, if user edits and makes major changes Mastodon kindly asks “You've put some heavy edits in this text, would you like to replace old status?” and if user agrees, it works just like redraft, if not— well, we just display error “Edited text can't contain major alterations.”

Also I like to think that it'd be funny for @MastodonProject to tweet “So we've got an edit button, why are you still on Twitter?”

**Comment by Gargron** [comment]
Duplicate of #6829
